core/tests/programs.py

```diff
@@ -92,3 +92,16 @@ def test_is_frozen_is_updated(self):
         ).values()[0]
 
         self.assertTrue(updated_program['is_frozen'])
+
+    def filter_by_queue(self):
```

Comment:
did this test run? i think tests have to start with "test"?

Reply:
`pipenv run test core.tests.ProgramsTestCase`

Reply:
Now that you mention it, I don't think it did. There was the same number of tests after I put this in. I thought that was strange, shrugged my shoulders and committed because it passed. Ok cool, I'll fix and resubmit.